### PR TITLE
changes to participant. Closes #416

### LINKED ISSUE #416: Attributes of an ids:Participant
The current Participant class has the following attributes:

https://w3id.org/idsa/core/version | http://www.w3.org/2001/XMLSchema#string | Version identifier of the Managed Entity
-- | -- | --
  | https://w3id.org/idsa/core/corporateEmailAddress | http://www.w3.org/2001/XMLSchema#string | Email address for contacting the participant on a general level.
  | https://w3id.org/idsa/core/corporateHomepage | http://www.w3.org/2001/XMLSchema#anyURI | General official homepage of the participant.
  | https://w3id.org/idsa/core/memberParticipant | https://w3id.org/idsa/core/Participant | Indicates that a participant has a member which  is again a participant. This is useful for defining hierarchical  relations in a participant's organization as well as identifying groups  of participants to capture, e.g., members of a collaboration.
  | https://w3id.org/idsa/core/participantRefinement | https://w3id.org/idsa/core/AbstractConstraint | Conditions which needs to be satisfied that a  single Participant can be seen as a member of the subject Participant.  For instance, all Participants with their headquarter in Europe might be  a potential consumer of GDPR-releated data.
  | https://w3id.org/idsa/core/memberPerson | https://w3id.org/idsa/core/Person | Indicates membership of a person to an organization.
  | https://w3id.org/idsa/core/participantCertification | https://w3id.org/idsa/core/ParticipantCertification | Certification issued for the given Participant.
  | https://w3id.org/idsa/core/primarySite | https://w3id.org/idsa/core/Site | Indicates a primary site for the Organization,  this is the default means by which an Organization can be contacted and  is not necessarily the formal headquarters.

That's not so much. We need to add a few more critical things, for instance VAT, 'Handelsregistereintrag', 'Gerichtssitz', responsible manager, ...

### PR DESCRIPTION
**Pull request by HaydarAk**

### CODE REVIEW

File: model/participant/Participant.ttl [hunk 1]
```diff
@@ -102,6 +112,38 @@ ids:participantRefinement
...
 
+ids:hasBusinessIdentifier
+	a owl:ObjectProperty ;
+	rdfs:domain ids:Participant ;
+	rdfs:range ids:BusinessIdentifier ;
+	rdfs:label "has business identifier"@en;
+	rdfs:comment "Company identifiers, described via ids:BusinessIdentifier."@en ;
+	.
+	
+ids:vatID
```
Comment (JohannesLipp) [inline model/participant/Participant.ttl:123]: Please be consistent with the property naming. Either add "hasProperty" or "property" to all (cf. `hasBusinessIdentifier` vs `vatID`), thanks!

Comment (JohannesLipp) [reply model/participant/Participant.ttl:123]: Also see `legalForm` and `jurisdiction` below.

File: model/participant/Participant.ttl [hunk 2]
```diff
@@ -102,6 +112,38 @@ ids:participantRefinement
...
+
+ids:legalForm
+	a owl:DatatypeProperty ;
+	rdfs:domain ids:Participant ;
+	rdfs:range xsd:string ;
+	rdfs:label "legal form"@en ;
+	rdfs:comment "Legal form of the participant. It is advisable to use an existing standard such as ISO-20275."@en ;
+.
+
+ids:jurisdication
```
Comment (JohannesLipp) [inline model/participant/Participant.ttl:139]: IMHO, the proper spelling is `jurisdiction` - please double-check and also update label and comment if required, thanks!

File: testing/participant/ParticipantShape.ttl [hunk 3]
```diff
@@ -70,6 +69,44 @@ shapes:ParticipantShape
...
+		sh:severity sh:Violation ;
+		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/participant/ParticipantShape.ttl> (ParticipantShape): An ids:hasBusinessIdentifier property must point from an ids:Participant to an ids:BusinessIdentifier."@en ;
+	];
+
+	sh:property [
+		a sh:PropertyShape ;
+		sh:path ids:vatID ;
+		sh:datatype xsd:string ;
+		sh:severity sh:Violation ;
+		#sh:minCount 1 ;
```
Comment (JohannesLipp) [inline testing/participant/ParticipantShape.ttl:86]: We should either remove this commented-out line, or add a proper comment to explain possible future development. Please also see the two following properties below.

**Review by JohannesLipp — CHANGES_REQUESTED** [review]
Minor changes, please revise these. Thanks!

**Comment by HaydarAk** [comment]
Alle suggestions should be resolved via 03969e0.
Could you re-check :) ?

**Review by JohannesLipp — APPROVED** [review]
Thank you for addressing all requested changes. Good work.

**Pull request merged**